File: src/main/java/com/rokucraft/rokubot/command/GuildIndependentCommand.java
package com.rokucraft.rokubot.command;

import net.dv8tion.jda.api.entities.Guild;
import net.dv8tion.jda.api.interactions.commands.build.CommandData;
import org.checkerframework.checker.nullness.qual.NonNull;

public interface GuildIndependentCommand extends Command {
    @Override
    default @NonNull CommandData getData(@NonNull Guild guild) {
        return getData();
    }

    @NonNull CommandData getData();
}

File: src/main/java/com/rokucraft/rokubot/command/GuildIndependentCommand.kt
package com.rokucraft.rokubot.command

import net.dv8tion.jda.api.entities.Guild
import net.dv8tion.jda.api.interactions.commands.build.CommandData

interface GuildIndependentCommand : Command {
    override fun getData(guild: Guild): CommandData {
        return getData()
    }

    fun getData(): CommandData
}
